retrieving/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	client, err := mongo.NewClient(options.Client().ApplyURI("<ATLAS_URI_HERE>"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	err = client.Connect(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Disconnect(ctx)
+
+	podcastsCollection := client.Database("quickstart").Collection("podcasts")
+	episodesCollection := client.Database("quickstart").Collection("episodes")
+
+	var podcast bson.M
+	err = podcastsCollection.FindOne(ctx, bson.M{}).Decode(&podcast)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(podcast)
+
+	cursor, err := episodesCollection.Find(ctx, bson.M{"duration": 25})
+	if err != nil {
+		log.Fatal(err)
+	}
+	var episodes []bson.M
+	if err = cursor.All(ctx, &episodes); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(episodes)
+
+	opts := options.Find()
+	opts.SetSort(bson.D{{"duration", -1}})
+	cursor, err = episodesCollection.Find(ctx, bson.D{{"duration", bson.D{{"$gt", 24}}}}, opts)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer cursor.Close(ctx)
+	for cursor.Next(ctx) {
```

Comment:
Why is `Next` used here but `All` is used for the example above? I think they should be consistent.